examples/opt_imports.rs

```diff
@@ -1,29 +1,29 @@
 use std::env;
 
 fn main() {
-	let args = env::args().collect::<Vec<_>>();
-	if args.len() != 3 {
-		println!("Usage: {} input_file.wasm output_file.wasm", args[0]);
-		return
-	}
+    let args = env::args().collect::<Vec<_>>();
+    if args.len() != 3 {
+        println!("Usage: {} input_file.wasm output_file.wasm", args[0]);
+        return;
+    }
 
-	// Loading module
-	let mut module = pwasm_utils::Module::from_elements(
-		&parity_wasm::deserialize_file(&args[1]).expect("Module deserialization to succeed"),
-	)
-	.expect("Failed to parse parity-wasm format");
+    // Loading module
+    let mut module = pwasm_utils::Module::from_elements(
```

Comment:
This doesn't compile for me (try `cargo-test`):

```
  --> examples/opt_imports.rs:13:30
   |
13 |     let mut module = Module::from_elements(
   |                              ^^^^^^^^^^^^^ function or associated item not found in `Module`
```

Reply:
I added back the mods I had removed. They should not be needed by `casper-execution-engine`, but let's be conservative and do this kind of clean-up later, to not put too many things in this unreadable commit.